android/build_component.sh

```diff
@@ -26,6 +27,9 @@ while [ $# -gt 0 ]; do
         component=$2
         shift
         ;;
+        --with-wrappture)
+        wrappture="$key"
```

Comment:
`wrappture` -> `build_wrappture`

Reply:
I change it --with-vcpkg
and build_with_vcpkg